src/containers/Home.tsx

```diff
@@ -34,17 +32,16 @@ class Home extends React.Component<{}, CreateGameState> {
     GameStore.createGame(token);
```

Comment:
Should this be awaited, or redirected after the promise resolves?

Reply:
I actually don't think it really matters -- post-redirect the app attaches the game listener and just waits to pull down the game *shrug*